root/components/ElectionTable/ElectionTableRows.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+const React = require('react');
+const {l, lp} = require('../../static/scripts/common/i18n');
+const EditorLink = require('../../static/scripts/common/components/EditorLink');
+const entityHref = require('../../static/scripts/common/utility/entityHref');
+const formatUserDate = require('../../utility/formatUserDate');
+
+const votesVisible = (election, user) => {
+  const involvedUser = user && (
+    election.proposer.id === user.id ||
+    election.candidate.id === user.id ||
+    (election.seconder1 && election.seconder1.id === user.id) ||
+    (election.seconder2 && election.seconder2.id === user.id)
+  );
+
+  if (election.isClosed || involvedUser) {
+    return true;
+  }
+  return false;
+};
+
+const ElectionTableRows = ({elections}) => (
+  elections.map(election => (
+    <tr className="odd" key={election.id}>
+      <td><EditorLink editor={election.candidate} /></td>
+      <td>{lp(election.shortStatusName, 'autoeditor election status (short)')}</td>
+      <td>{formatUserDate($c.user, election.proposedTime)}</td>
```

Comment:
I am having a bit of a problem with this function. Not sure if I am using this correctly. @mwiencek any help?

Reply:
NVM, fixed it.